Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ghcr.io/kbase/cdm-spark-standalone:pr-32
```

Comment:
I might do something like `test.Dockerfile`.

Reply:
Hmm... if there was going to be more than one Dockerfile I'd do that, but if you don't use the standard name then you have to provide the name to `docker build .`